dcs/flyingunit.py

```diff
@@ -97,6 +97,8 @@ def reset_loadout(self):
 
     def set_default_preset_channel(self, freq: float) -> None:
         """Sets the frequency for channel 1 of the main radio."""
+        if self.radio is None:
+            return
```

Comment:
This is for setting channels, so probably should raise, and the callers should have the check?

Reply:
Sure!